docs/fast_data/data_catalog/overview.md

```diff
@@ -0,0 +1,45 @@
+---
+id: overview_data_catalog
+title: Data Catalog Overview
+sidebar_label: Overview
+---
+
+:::caution
+Please notice that the information explained in the following page, regarding the deployment of a Data Catalog Agent,
+are valid only for Console projects adopting the [Enhanced Workflow](/development_suite/set-up-infrastructure/enhanced-project-workflow.md).
+
+The Data Catalog Agent can not be deployed in projects adopting the older Console workflow.
+:::
+
+A Mia-Platform Console Project can be configured to function as a Data Catalog, that is a centralized 
+location where it is possible to collect metadata concerning all the data assets of an organization.
+Within the project it is possible to set up our Data Catalog Agent, which can be configured to connect to different sources,
+such as:
+
+- Oracle Database
+- Postgres Database
+- MySQL Database
+- Microsoft SQL Server Database
```

Comment:
@danibix95 are we supporting everything already? I'm not sure about mysql and mssql, can you pls confirm? Otherwise we can put (coming soon)

Reply:
I can confirm that they are already implemented. The supported versions for each database are stated in the specific marketplace item documentation, which will be release soon.